### PR TITLE
Add release notes 0.2.0

### LINKED ISSUE #1795: Release v0.2.0
**What version number is to be released?**
v0.2.0

**What is the estimated date for this release?**
06.02.2023

**Release procedure**
Please refer to the [Release Procedure doc](https://github.com/VirtusLab/scala-cli/blob/main/.github/release/release-procedure.md).

**Release notes**
Please remember to create a pull request with a draft of the release notes in the [Release Notes History doc](https://github.com/VirtusLab/scala-cli/blob/main/website/docs/release_notes.md).
Please make sure the notes render correctly on [the website](https://scala-cli.virtuslab.org/docs/release_notes) - that includes swapping out GitHub-idiomatic @mentions of users, links to PRs, issues, etc.

### PR DESCRIPTION
**Pull request by lwronski**

### CODE REVIEW

File: website/docs/release_notes.md [hunk 1]
```diff
@@ -7,6 +7,133 @@ import ReactPlayer from 'react-player'
 
 # Release notes
 
+## [v0.2.0](https://github.com/VirtusLab/scala-cli/releases/tag/v0.2.0)
+
+### Change ScalaCLI default behavior
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
### Require the `--power` option for restricted features by default
```

File: website/docs/release_notes.md [hunk 2]
```diff
@@ -7,6 +7,133 @@ import ReactPlayer from 'react-player'
...
+
+By default, now ScalaCLI limited some functionalities, that means that some commands or options may stop working. However, restricted features can be accessed by using the `--power` flag. Alternatively, the `power` mode can be turned on globally by running:
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
````suggestion
Until now, Scala CLI has been limiting some of its functionalities in its `scala` distribution.
Starting with `v0.2.0`, those limitation will be applied to all distributions, including `scala-cli`.

Restricted features can be accesed by using the `--power` launcher flag. Do note that launcher flags have to be passed **before** the sub-command.
```bash ignore
scala-cli --power package .
```

Alternatively, the `power` mode can be turned on globally by running:
````

File: website/docs/release_notes.md [hunk 3]
```diff
@@ -7,6 +7,133 @@ import ReactPlayer from 'react-player'
...
+```bash ignore
+scala-cli config power true 
+```
+
+Please note that this change may affect your existing scripts or workflows that rely on the limited commands from ScalaCLI (such as `package`, `publish`). You can still use this command with enabled `power` mode.
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
Please note that this change may affect your existing scripts or workflows that rely on the limited commands from ScalaCLI (such as `package`, `publish`). You can still use those commands with `power` mode enabled.
```

File: website/docs/release_notes.md [hunk 4]
```diff
@@ -7,6 +7,133 @@ import ReactPlayer from 'react-player'
...
+#    scala-cli config power true
+$ scala-cli config power true
+$ scala-cli package Hello.scala
+# Wrote Hello, run it with
+#   ./Hello
+```
+
+Added by [@lwronski](https://github.com/lwronski) in https://github.com/VirtusLab/scala-cli/pull/1835 and  https://github.com/VirtusLab/scala-cli/pull/1849
+
+### Allow executable Scala scripts without file suffix
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
### Allow executable Scala scripts without a file extension
```

File: website/docs/release_notes.md [hunk 5]
```diff
@@ -7,6 +7,133 @@ import ReactPlayer from 'react-player'
...
+
+With this release, Scala scripts can now be executed without a file suffix `.sc` when using the `shebang` command.
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
As of this release Scala scripts without the `*.sc` file extension will be supported for execution when using the `shebang` command.
```

File: website/docs/release_notes.md [hunk 6]
```diff
@@ -7,6 +7,133 @@ import ReactPlayer from 'react-player'
...
+println(args.size)
+println(args.headOption)
+```
+```bash
+chmod +x hello
+./hello Hello World
+#2
+#Some(Hello)
+ ```
+Note that files with no extensions are always run as scripts even though they may contain e.g. valid `.scala` program.
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
Note that files with no extension are always run as scripts even though they may contain e.g. a valid `.scala` program.
Also, do note that this feature has only been added for `shebang` - the `run` sub-command (which is the default way of running inputs when a sub-command is not specified explicitly) will not support this.
```

File: website/docs/release_notes.md [hunk 7]
```diff
@@ -7,6 +7,133 @@ import ReactPlayer from 'react-player'
...
+
+Added by [@MaciejG604](https://github.com/MaciejG604) in https://github.com/VirtusLab/scala-cli/pull/1802
+
+###  Breaking changes
```
Comment (Gedochao) [inline website/docs/release_notes.md]: technically speaking, the global `--power` is also a breaking change...
```suggestion
###  Other breaking changes
```

File: website/docs/release_notes.md [hunk 8]
```diff
@@ -7,6 +7,133 @@ import ReactPlayer from 'react-player'
...
+
+#### Remove ammonite imports support
+Based on discussion, support for `$ivy` and `$dep` ammonite imports has been removed. To convert `$ivy` and `$dep` imports into the `using dep` directive, you can use the provided actionable action.
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
The support for `$ivy` and `$dep` ammonite imports has been removed. To easily convert existing 
 `$ivy` and `$dep` imports into the `using dep` directive in your sources, you can use the provided actionable diagnostic.
```

Comment (Gedochao) [reply website/docs/release_notes.md]: NIT maybe even a gif with the diagnostic could be useful?

File: website/docs/release_notes.md [hunk 9]
```diff
@@ -7,6 +7,133 @@ import ReactPlayer from 'react-player'
...
+
+Removed by [@MaciejG604](https://github.com/MaciejG604) in https://github.com/VirtusLab/scala-cli/pull/1787
+
+#### Drop metabrowse command
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
#### Drop the `metabrowse` sub-command
```

File: website/docs/release_notes.md [hunk 10]
```diff
@@ -7,6 +7,133 @@ import ReactPlayer from 'react-player'
...
+
+With this release, support for Metabrowse has been removed from Scala CLI. This change was made in order to limit the number of features that we need to support, especially since the `Metabrowse` project is no longer being actively worked on.
+
+Remove by [@lwronski](https://github.com/lwronski) in https://github.com/VirtusLab/scala-cli/pull/1867
+
+#### Rename `using lib` to `using dep`
+
+To be more consistent with dependency command line options `--dep`, the dependency using directive is now passed by `using dep`.
+Please note that we have kept the alias of the old directive (`lib`, `libs`) for backwards compatibility.
```
Comment (Gedochao) [inline website/docs/release_notes.md:117]: We keep the backwards compatibility, so it's not actually a breaking change.

File: website/docs/release_notes.md [hunk 11]
```diff
@@ -7,6 +7,152 @@ import ReactPlayer from 'react-player'
 
 # Release notes
 
+## [v0.2.0](https://github.com/VirtusLab/scala-cli/releases/tag/v0.2.0)
+
+### Require the `--power` option for restricted features by default
+
+Until now, Scala CLI has been limiting some of its functionalities in its `scala` distribution.
+Starting with `v0.2.0`, those limitation will be applied to all distributions, including `scala-cli`.
```
Comment (tgodzik) [inline website/docs/release_notes.md:15]:
```suggestion
Starting with `v0.2.0`, those limitation will be applied to all distributions, including `scala-cli`. This was done in order to make the behaviour consistent between the future Scala runner and ScalaCLI.
```
some explanation maybe?

File: website/docs/release_notes.md [hunk 12]
```diff
@@ -7,6 +7,152 @@ import ReactPlayer from 'react-player'
...
+
+Also, do note that this feature has only been added for `shebang` - the `run` sub-command (which is the default way of running inputs when a sub-command is not specified explicitly) will not support this.
+
+Added by [@MaciejG604](https://github.com/MaciejG604) in [#1802](https://github.com/VirtusLab/scala-cli/pull/1802)
+
+### Export information about the project in the json format
+
+```bash ignore
+scala-cli --power export --json
+```
```
Comment (tgodzik) [inline website/docs/release_notes.md:75]: Could we should an example output here? OR part of it?

Comment (lwronski) [reply website/docs/release_notes.md:75]: Yeah, so I added basic json output without `Resolvers` field.

File: website/docs/release_notes.md [hunk 13]
```diff
@@ -7,6 +7,193 @@ import ReactPlayer from 'react-player'
...
+
+### Export Project configuration to Json
+
+It is now possible to export configuration from Scala CLI project to Json format with the `export` sub-command.
+
+```bash ignore
+scala-cli --power export --json .
+```
+
+It is currently exporting basic information about the project and includes, for example, the following field:
```
Comment (MaciejG604) [inline website/docs/release_notes.md]:
```suggestion
It is currently exporting basic information about the project and includes, for example, the following fields:
```

File: website/docs/release_notes.md [hunk 14]
```diff
@@ -7,6 +7,193 @@ import ReactPlayer from 'react-player'
 
 # Release notes
 
+## [v0.2.0](https://github.com/VirtusLab/scala-cli/releases/tag/v0.2.0)
+
+### Require the `--power` option for restricted features by default
+
+Until now, Scala CLI has been limiting some of its functionalities in its `scala` distribution.
+Starting with `v0.2.0`, those limitation will be applied to all distributions, including `scala-cli`. This was done in order to make the behaviour consistent between the future Scala runner and ScalaCLI.
```
Comment (Gedochao) [inline website/docs/release_notes.md]:
```suggestion
Starting with `v0.2.0`, those limitation will be applied to all distributions, including `scala-cli`. 

This was done in order to make the behaviour consistent with Scala CLI acting as the Scala runner.
```
Missed this spot earlier 😅

File: website/docs/release_notes.md [hunk 15]
```diff
@@ -7,6 +7,193 @@ import ReactPlayer from 'react-player'
...
+```scala compile
+ //> using dep "org.scalameta::munit:0.7.29"
+```
+Renamed by [@lwronski](https://github.com/lwronski) in [#1827](https://github.com/VirtusLab/scala-cli/pull/1827)
+
+### Other breaking changes
+
+#### Remove ammonite imports support
+The support for `$ivy` and `$dep` ammonite imports has been removed. 
+To easily convert existing `$ivy`   and `$dep` imports into the `using dep` directive in your sources, you can use the provided actionable diagnostic.
```
Comment (MaciejG604) [inline website/docs/release_notes.md]:
```suggestion
To easily convert existing `$ivy` and `$dep` imports into the `using dep` directive in your sources, you can use the provided actionable diagnostic.
```

**Review by MaciejG604 — APPROVED** [review]

**Comment by tgodzik** [comment]
Do we want to merge the help PR first @Gedochao  ?

**Review by tgodzik — APPROVED** [review]
Beside the above comment LGTM!

**Comment by Gedochao** [comment]
> Do we want to merge the help PR first @Gedochao ?

Yup, that is the plan I think

**Review by Gedochao — APPROVED** [review]

**Pull request merged**